### ISSUE TITLE
Allow PWs to be linked to base-64 encoded image

### ISSUE THREAD

**Issue by DeniseSl22** [wikipathways/cytoscape-wikipathways-app#112]
Since the String app visualises protein structures in the nodes from base-64 encoded images, we were hoping we could also use similar images to visualise in PWs. This would allow us (and collaborators) to add one column to a PW/network (with the Cytoscape API), and then use this column to perform a passthrough mapping so we can visualise metabolomics imaging data on the PWs (as automated as possible).

We've also tried working with the image library in Cytoscape, but the option to automatize this with the Cytoscape API has vanished :(. We don't want to add all the images manually in the library first; the steps I mentioned above will be part of a large automated workflow (so really important to take several project forward).

As an example dataset, see below, which should work on the [KrebsCycle PW for humans](https://www.wikipathways.org/index.php/Pathway:WP78) (which is only visualised nicely as a network I saw, probably due to all the curved lines in the PW drawing).

[Example_network_EncodedImages.txt](https://github.com/wikipathways/cytoscape-wikipathways-app/files/4102368/Example_network_EncodedImages.txt)

**Comment by DeniseSl22** [comment]
If there would be a size restriction for the images (in pixels), please let me know, that's something we could work with. The encoded images are now black/white (I believe), not in colour (as the protein structures from String are). That's something we could work on later on our side (don't think it should cause any issues). And if there are other requirements that should be met on our side for this to work, please let us know :).

**Comment by AdamStuart** [comment]
as I understand this we can add an attribute inside the Graphics element:

<Graphics width="100"   ...      img="9084230498...." />

What I was proposing was: 

<Graphics width="100"   ...      img="http://wp.nl.org/1243" />

The xml nerds would want to add this as a subelement, not an attribute, but this approach seems more consistent with GPML

**Comment by mkutmon** [comment]
Importantly, the images should not be included in the GPML. They are an overlay after the pathway is opened in Cytoscape. It's mainly a visual style aspect and we tried to mimic the stringApp with that column containing the base64 encoded image but for some reason that doesn't work with the WikiPathways style. 
Maybe we need a short Skype call on this. I'll also try to send you what I already tried to give you an idea of what we want to achieve.

**Comment by DeniseSl22** [comment]
Yes, agree with @mkutmon ; this image visualisation would be an additional step in data analysis, which we want to perform in Cytoscpae (not in PV).
